apps/server/src/shared/domain/entity/user.entity.ts

```diff
@@ -71,9 +71,8 @@ export class User extends BaseEntityWithTimestamps implements IEntityWithSchool
 	@Index()
 	ldapDn?: string;
 
-	@Property({ nullable: true })
-	@Index()
-	ldapId?: string;
+	@Property({ nullable: true, fieldName: 'ldapId' })
+	externalId?: string;
```

Comment:
Why you removed the index here? And please add doc for this field. What is that and why we renamed it

Reply:
index restored and doc written